Makefile

```diff
@@ -181,6 +187,11 @@ deploy-azero: # Deploy azero contracts
 deploy-azero: compile-azero
 	cd azero && AZERO_ENV=$(AZERO_ENV) npm run deploy
 
+.PHONY: upload-azero
+upload-azero: # Upload azero contract code without instantiating (useful for upgrades)
+upload-azero: compile-azero
+	cd azero && AZERO_ENV=$(AZERO_ENV) npm run upload
```

Comment:
This doesn't run verifiable compilation, or am I wrong?

If not then it probably should - in preparation for the upgrade.